src/test/mir-opt/const_allocation2.rs

```diff
@@ -1,3 +1,5 @@
+// only-x86_64
```

Comment:
I wonder if we could just use revisions and have one for 32-bit and one for 64-bit targets?
It would mean duplicating the output files, but maybe we *should* be that thorough *anyway*?

Reply:
I didn't suggest this on purpose :D I don't want to implement it (I can't test 32 bit, I just get compilation failures if I add `--target i686-unknown-linux-gnu`

Reply:
Ah I forgot, I had to install `gcc-multilib` to get the 32bit builds to work.

Reply:
Btw, in case it's unclear: I don't want the test to switch targets, but rather for the revision that doesn't apply to whatever the current target is, to be ignored, so effectively one of the two possible outputs is used for any given target.

Reply:
yes I get that, but I can't test it so I can't really implement it. I'll try to setup cross compiling and then implement it